src/hadoop-resource-manager/deploy/hadoop-resource-manager.yaml.template

```diff
@@ -32,9 +32,34 @@ spec:
         prometheus.io/path: '/metrics'
         prometheus.io/port: '{{ cluster_cfg[ "hadoop-resource-manager" ]["yarn_exporter_port"] }}'
     spec:
+      hostAliases:
+      - ip: "{{ cluster_cfg[ "hadoop-resource-manager" ][ "master-ip" ] }}"
+        hostnames:
+        - "{{ cluster_cfg[ "hadoop-resource-manager" ][ "master-ip" ] }}"
+      - ip: "{{ cluster_cfg[ "hadoop-jobhistory" ][ "timeline-server-ip" ] }}"
+        hostnames:
+        - "{{ cluster_cfg[ "hadoop-jobhistory" ][ "timeline-server-ip" ] }}"
       hostNetwork: true
       hostPID: false
       containers:
+      - name: yarn-exporter
```

Comment:
do not make unnecessary code move, git blame will not be able to see the real history.

Reply:
done